modules/login/templates/form_passwordexpired.tpl

```diff
@@ -35,3 +35,4 @@
     </div>
```

Comment:
Can't comment directly on the file,  but you renamed  `htdocs/lost_password.php → ...les/login/templates/form_lostpassword.tpl` aka a php file into a tpl file and that doesn't look right